app/src/main/java/fr/gaulupeau/apps/Poche/ui/MainActivity.java

```diff
@@ -229,6 +236,10 @@ private void updateTime() {
         EventBus.getDefault().register(this);
     }
 
+    private boolean isCurrentThemeDark() {
```

Comment:
Sorry, I meant to have this into `fr.gaulupeau.apps.Poche.ui.Themes` as a public function instead.

Reply:
Thanks, I hope I've did it right this time.